msim/src/sim/net/mod.rs

```diff
@@ -80,7 +80,11 @@ struct PlaceholderFileDes(libc::c_int);
 impl Drop for PlaceholderFileDes {
     fn drop(&mut self) {
         unsafe {
-            bypass_close(self.0);
+            debug!("closing {:?}", self);
+            let stat = bypass_close(self.0);
+            if stat != 0 {
```

Comment:
Clarifying my understanding - so this is the return value from calling the C syscall `close`?

Reply:
good point, i should be logging errno